cmd/state-svc/internal/resolver/analytics/customdimensions.go

```diff
@@ -29,5 +29,6 @@ func (d *customDimensions) toMap(projectName, output, userID string) map[string]
 		"11": d.sessionToken,
 		"12": d.uniqID,
 		"13": d.updateTag,
+		"14": projectID,
```

Comment:
@Naatan I think this needs to be added to GA

Reply:
Done